static/app/components/workflowEngine/gridCell/actionCell.tsx

```diff
@@ -35,20 +36,29 @@ export function ActionCell({actions}: ActionCellProps) {
   }
   const actionsList = actions.map(action => ActionMetadata[action].name).join(', ');
   return (
-    <Flex align="center" gap={space(0.75)}>
+    <ActionContainer align="center" gap={space(0.75)}>
       <IconContainer>
         <IconCircledNumber number={actions.length} />
       </IconContainer>
-      <Tooltip title={actionsList}>
+      <Tooltip title={actionsList} disabled={disabled}>
         <ActionsList>{actionsList}</ActionsList>
       </Tooltip>
-    </Flex>
+    </ActionContainer>
   );
 }
 
+const ActionContainer = styled(Flex)`
+  /* overflow: hidden;
+  white-space: nowrap; */
+`;
+
 const ActionsList = styled('span')`
   ${p => p.theme.tooltipUnderline()};
   text-overflow: ellipsis;
+  display: flex;
+  overflow: hidden;
+  white-space: nowrap;
+  max-width: 100%;
```

Comment:
really struggling with getting the ellipsis to appear when there's overflow...

I think the list needs to be wrapped in a `div` that has `display: inline-block` but when I wrap the full cell component it breaks the rest of the layout 😭

Reply:
Hmm yeah we are going to need a better solution for this. I personally don't think we should block this PR on getting robust cell truncation working. I'll have to block out some time to explore different solutions, probably using [`ResizeObserver`](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver) or [Container Queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries).